src/Entities/Family.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Choredo\Entities;
+
+use Ramsey\Uuid\Uuid;
+
+/**
+ * Class Family
+ * @package Choredo\Entities
+ *
+ * @ORM\Entity
+ */
+class Family
+{
+    /**
+     * @var Uuid
+     *
+     * @ORM\Id
+     * @ORM\Column(type="uuid")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="payment_strategy")
+     */
+    private $paymentStrategy;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true, name="completion_threshold")
+     */
+    private $completionThreshold;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="smallint", name="week_start_day")
+     */
+    private $weekStartDay;
+
+    /**
+     * @return Uuid
+     */
+    public function getId(): Uuid
+    {
+        if (!isset($this->id)) {
+            $this->id = Uuid::uuid4();
+        }
+        return $this->id;
+    }
+
+    /**
+     * @param Uuid $id
+     * @return Family
+     */
+    public function setId(Uuid $id): Family
```

Comment:
Personally I very much dislike setters and prefer constructor instantiation. It allows for less chance for an invalid/partial state for the objects.